openreview/journal/journal.py

```diff
@@ -403,7 +408,8 @@ def notify_readers(self, edit, content_fields=[]):
         ## Notify action editors
         if is_public or self.get_action_editors_id(number=forum.number) in readers:
             message = f'''Hi {{{{fullname}}}},
-{before_invitation} {lower_formatted_invitation} {action} on a submission for which you are serving as Action Editor {forum.content['title']['value']}
+
+{before_invitation} {lower_formatted_invitation} on a submission you are an Action Editor for has been {action}
```

Comment:
(This sounds slight weird to me) Just a suggestion:

> {before_invitation} {lower_formatted_invitation} has been {action} on a submission for which you are an Action Editor

Reply:
The change is a suggestion from @zbialecki , I like your suggestion tough

Reply:
I changed the text with Celeste' suggestion, it is ready to merge now.